# KernelSVM

In [1]:
from kernelsvr import main
main()

 processing train and test data by batch method took 1.357150 s
the training of linear svr model took 51.941256399964914s.
explained_variance_score: 0.9935008122276743
final mse:  0.018613462623325337


TypeError: 'NoneType' object is not iterable

<img src = "reports/svm/svr_kernal_scaled.png" ,width = 500 ,height = 500>

### Results 
    the training of linear svr model took 51.941256399964914s.
    explained_variance_score: 0.9935008122276743
    final mse:  0.018613462623325337